shims/spark320/src/main/scala/com/nvidia/spark/rapids/shims/spark320/Spark320Shims.scala

```diff
@@ -483,15 +485,40 @@ class Spark320Shims extends Spark32XShims {
             " the Java process and the Python process. It also supports scheduling GPU resources" +
             " for the Python process when enabled.",
         ExecChecks(TypeSig.commonCudfTypes, TypeSig.all),
-        (aggPy, conf, p, r) => new GpuAggregateInPandasExecMeta(aggPy, conf, p, r))
+        (aggPy, conf, p, r) => new GpuAggregateInPandasExecMeta(aggPy, conf, p, r)),
+      GpuOverrides.exec[BatchScanExec](
+        "The backend for most file input",
+        ExecChecks(
+          (TypeSig.commonCudfTypes + TypeSig.STRUCT + TypeSig.MAP + TypeSig.ARRAY +
+            TypeSig.DECIMAL_64).nested(),
+          TypeSig.all),
+        (p, conf, parent, r) => new SparkPlanMeta[BatchScanExec](p, conf, parent, r) {
+          override val childScans: scala.Seq[ScanMeta[_]] =
+            Seq(GpuOverrides.wrapScan(p.scan, conf, Some(this)))
+
+          override def tagPlanForGpu(): Unit = {
+            // Only Scan with SupportsRuntimeFiltering can support DPP.
+            if (!p.runtimeFilters.isEmpty) {
+              willNotWorkOnGpu("The DPP has not been supported")
```

Comment:
Can we update the comment to be a little more clear? Perhaps something like.

"Runtime filtering (DPP) on datasource V2 is not supported"